coderd/workspaces_test.go

```diff
@@ -867,3 +1067,11 @@ func mustLocation(t *testing.T, location string) *time.Location {
 
 	return loc
 }
+
+func must[T any](s T, err error) T {
+	if err != nil {
+		panic(err)
+	}
+
+	return s
+}
```

Comment:
Because this is only used once, I'd rather us `require.NoError` it for now. I'm not opposed entirely, but I'd rather introduce this in a way that's consistent with the rest of the codebase, not just used in one spot.

Reply:
The `cryptorand` package should really have `MustIntn` and other funcs. Without that, you cannot use `cryptorand` inside struct initialization, like `testCases`.

`require.NoError` does not help in this case. I'll just used a fixed index.

Reply:
Cryptographic randomness isn't guaranteed on Mac and Windows, which is why they were removed. It's not safe to use in production, and could cause a panic.

Reply:
It's in a unit test. Not production

Reply:
Maybe a `cryptorandtest` pkg... unsure. Just hurts readability to have to instantiate vars outside the `testCases` structs.